<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventilate_newuser</class>
 <widget class="QDialog" name="ventilate_newuser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventilate: New User</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(66, 66, 66);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>7</number>
   </property>
   <property name="leftMargin">
    <number>7</number>
   </property>
   <property name="topMargin">
    <number>7</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>7</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="minimumSize">
      <size>
       <width>461</width>
       <height>84</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>84</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>461</width>
       <height>84</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/WelcometoVentilate.svg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="vspControlsTop">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>155</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlUsernameLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <item>
      <widget class="QLabel" name="lblUsername">
       <property name="minimumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>Username: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lnedUsername">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="maxLength">
        <number>64</number>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblUsernameContextInfo">
       <property name="minimumSize">
        <size>
         <width>133</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>133</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>133</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspUsername">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlPasswordLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <item>
      <widget class="QLabel" name="lblPassword">
       <property name="minimumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>Password: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lnedPassword">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspPassword">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlConfirmPasswordLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <item>
      <widget class="QLabel" name="lblConfirmPassword">
       <property name="minimumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>Confirm Password: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lnedConfirmPassword">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblConfirmPasswordContextInfo">
       <property name="minimumSize">
        <size>
         <width>133</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>133</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>133</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspConfirmPassword">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlEmailLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <item>
      <widget class="QLabel" name="lblEmail">
       <property name="minimumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>Email: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lnedEmail">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspEmail">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlConfirmEmailLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <item>
      <widget class="QLabel" name="lblConfirmEmail">
       <property name="minimumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>98</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255,255,255);</string>
       </property>
       <property name="text">
        <string>Confirm Email: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lnedConfirmEmail">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>140</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frame">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblConfirmEmailContextInfo">
       <property name="minimumSize">
        <size>
         <width>133</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>133</width>
         <height>21</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>133</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspConfirmEmail">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vspControlsBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>155</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ventilate_newuser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ventilate_newuser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
